alws/crud.py

```diff
@@ -186,6 +187,15 @@ async def get_available_build_task(
     return db_task
 
 
+async def update_failed_build_items(db: Session, build_id: int):
+    query = sqlalchemy.and_(models.BuildTask.build_id == build_id, 
+                            models.BuildTask.status == BuildTaskStatus.FAILED)
+    async with db.begin():
+        await db.execute(update(models.BuildTask).where(query).values(
+            status=BuildTaskStatus.IDLE))
+        await db.commit()
```

Comment:
Please, also include right build order for restarted items, you can check `alws/build_planner.py`
Right now, if you have two failed items in your build, after restart they will be run together